client/dashboard/components/truncate/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { __experimentalTruncate as WPTruncate, Tooltip } from '@wordpress/components';
+import { useMergeRefs, useResizeObserver } from '@wordpress/compose';
+import { useState, forwardRef } from 'react';
+
+export interface TruncateProps extends React.ComponentProps< typeof WPTruncate > {
+	tooltip: string;
+}
+
+function UnforwardedTruncate(
+	{ tooltip, numberOfLines = 0, ...props }: TruncateProps,
```

Comment:
Question: Is the `tooltip` prop necessary? Seems that it'd always be the same as the children.

Reply:
On second thought, maybe it's okay to keep it in case children has HTML markup. In that situation we might only want to pass raw text to the tooltip.